README.md

```diff
@@ -7,7 +7,23 @@ This challenge enables the Process Analytics team working at Bonitasoft to find
 
 You work on a new Open Source project as a frontend developer. This project is destined to the Bingo players.
 
-Your mission, if you accept it, is to create a new library and a demonstrator of this library to permit the Bingo players to display a new grid with random numbers, add a new token on a square, and change the background & add an interaction when a row/column is completed.
+Your mission, if you accept it, is to create a new library, and a demonstrator of this library to permit the Bingo players to display a new grid with random numbers, add a new token on a square, and change the background & add an interaction when a row/column is completed.
+
+## Bingo rules
+### Presentation of the game
+Bingo is a game of chance played with 90 balls, numbered from 1 to 90, where the drawn numbers are announced one after the other.  \
```

Comment:
❓ If you took the following or inspired from a site, could you add a link to it for more details? Otherwise, this is fine like this.

Reply:
There is different rules. I mixed them to simplify the implementation.
I started from the American rules, but in thoses rules, the numbers go to 75 and the centered square is a Star.